python/raft-ann-bench/src/raft-ann-bench/run/__main__.py

```diff
@@ -154,6 +154,7 @@ def run_build_and_search(
 ):
     for executable, ann_executable_path, algo in executables_to_run.keys():
         # Need to write temporary configuration
+        print("ALGO: %s" % algo)
```

Comment:
Do we want to always print this? Might be good to do logging (maybe future PR?)

Reply:
Good catch. I just removed it. Didn't realize that was still in there